Source/Core/TileAvailability.js

```diff
@@ -1,17 +1,17 @@
 /*global define*/
 define([
+    './DeveloperError',
```

Comment:
So close haha! There are still changes in this file, since things are slightly different. https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file may help